File: wallet/src/main/java/com/worldventures/wallet/ui/settings/help/feedback/SendFeedbackScreen.java
package com.worldventures.wallet.ui.settings.help.feedback;

import com.worldventures.wallet.ui.settings.help.feedback.base.BaseFeedbackScreen;

import rx.Observable;

public interface SendFeedbackScreen extends BaseFeedbackScreen {

   void applyFeedbackType(FeedbackType feedbackType);

   Observable<CharSequence> getTextFeedbackMessage();

   FeedbackType getFeedbackType();
}

File: wallet/src/main/java/com/worldventures/wallet/ui/settings/help/feedback/regular/SendFeedbackScreen.kt
package com.worldventures.wallet.ui.settings.help.feedback.regular

import com.worldventures.wallet.ui.settings.help.feedback.base.BaseFeedbackScreen

import rx.Observable

interface SendFeedbackScreen : BaseFeedbackScreen {

   val feedbackMessage: String

   val textFeedbackMessage: Observable<CharSequence>

   val feedbackType: FeedbackType

   fun applyFeedbackType(feedbackType: FeedbackType)
}
